openverse_catalog/dags/data_refresh/data_refresh_types.py

```diff
@@ -21,24 +21,24 @@ class DataRefresh:
 
     Optional Constructor Arguments:
 
-    default_args:      dictionary which is passed to the airflow.dag.DAG
-                       __init__ method.
-    start_date:        datetime.datetime giving the
-                       first valid execution_date of the DAG.
-    schedule_interval: string giving the schedule on which the DAG should
-                       be run.  Passed to the airflow.dag.DAG __init__
-                       method.
-    execution_timeout: datetime.timedelta giving the amount of time a given data
-                       pull may take.
-    doc_md:            string which should be used for the DAG's documentation markdown
+    default_args:         dictionary which is passed to the airflow.dag.DAG
+                          __init__ method.
+    start_date:           datetime.datetime giving the
+                          first valid execution_date of the DAG.
+    schedule_interval:    string giving the schedule on which the DAG should
+                          be run.  Passed to the airflow.dag.DAG __init__
+                          method.
+    data_refresh_timeout: datetime.timedelta giving the amount of time a given data
+                          pull may take.
+    doc_md:               string used for the DAG's documentation markdown
     """
 
     dag_id: str = field(init=False)
     media_type: str
     start_date: datetime = datetime(2020, 1, 1)
-    execution_timeout: timedelta = timedelta(hours=24)
     schedule_interval: Optional[str] = "@weekly"
     default_args: Optional[Dict] = field(default_factory=dict)
+    data_refresh_timeout: timedelta = 24 * 60 * 60  # 1 day
```

Comment:
This is no longer a timedelta, should it be? Since this is a dataclass, we can always accept an `int` and change it to a `timedelta` in the `__post_init__` function.

Reply:
You're right! It should not be a timedelta, the `timeout` on `HttpSensor` accepts an `int`. I'll update the type to match